SpaceCadetPinball/control.cpp

```diff
@@ -4081,7 +4133,11 @@ void control::SecretMissionGreenController(MessageCode code, TPinballComponent*
 	if (sink2 == caller)
 	{
 		lite198->MessageField = 1;
-		MissionControl(MessageCode::ControlMissionComplete, nullptr);
+		if (pb::FullTiltMode)
+		{
+			table_set_replay(4.0);
+		}
+	MissionControl(MessageCode::ControlMissionComplete, nullptr);
```

Comment:
too little indentation in this line again. what's happening here? is your IDE acting up?